src/index.html

```diff
@@ -11,6 +11,44 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      class="form"
+      action="/"
+      method="get">
+      <div class="searchBar big-searchBar" data-qa="big" >
+        <img
+          class="search__swg"
+          src="./images/Search.svg"
+          alt="search image"
+        >
+        <label class="lable-input">
+          <input
+            class="input__field"
+            placeholder="Try “Los Angeles”"
+            type="text"
+            name="bigSearchField"
+            id="bigSearchField"
+            data-qa="keypress"
+          >
+        </label>
+      </div>
+
+      <div class="searchBar small-searchBar" data-qa="small">
```

Comment:
It should be separate form for EACH input

Reply:
i don`t anderstund you